lib/v2/sduwh/news.js

```diff
@@ -0,0 +1,37 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const data = require('./data').news;
+const extractor = require('./extractor');
+
+module.exports = async (ctx) => {
+    const column = ctx.params.column ?? 'xyyw';
+    const baseUrl = data.url;
+    const response = await got.get(baseUrl + data.columns[column].url);
+    const $ = cheerio.load(response.data);
+    const items = $('.n_newslist li');
+    const out = await Promise.all(
+        items.map(async (index, item) => {
+            item = $(item);
+            const anchor = item.find('a');
+            const title = anchor.attr('title');
+            const href = anchor.attr('href');
+            const link = href.startsWith('http') ? href : baseUrl + '/' + href;
+            const { description, author } = await ctx.cache.tryGet(link, async () => await extractor(link, ctx));
+            const span = item.find('span');
+            const pubDate = new Date(span.text());
```

Comment:
```suggestion
            const pubDate = parseDate(span.text(), 'YYYY/MM/DD');
```
https://docs.rsshub.app/joinus/pub-date.html#shi-yong-gong-ju-lei-parse-date

---
Btw articles have the exact pubDate and time inside `.news_tit`, consider using it.